src/common/navigation.yaml

```diff
@@ -17,6 +17,9 @@ sections:
 
   - title: Technology
     pages:
+#      - path: /pox
```

Comment:
let's remove this comment, assuming it's a leftover from before. with that, we should be good to go